composer.json

```diff
@@ -42,7 +42,7 @@
         "symfony/property-info": "^3.4.30|^4.3.3",
         "symfony/phpunit-bridge": "^4.2",
         "symfony/twig-bridge": "^3.4|^4.1",
-        "twig/twig": "^1.34|^2.4",
+        "twig/twig": "^1.34|^2.12",
```

Comment:
Is this really necessary? I mean, composer will automatically pick 2.12 since it's the latest stable version.

Reply:
I can add a `prefer-lowest` build on PHP 7.4 if you'd like. I'm all for this change :)

Reply:
> I can add a `prefer-lowest` build on PHP 7.4 if you'd like.

If you do this, composer will install Twig 1.34 – with or without this change. 😉 Also, you will need to bump quite a few more packages to get the build green.

> I'm all for this change :)

I wouldn't oppose it. All I'm saying is that I don't see the need for this change because this library isn't suddenly incompatible with Twig 2.4 - 2.11.

Running a green build on bleeding-edge php with `prefer-lowest` dependencies shouldn't be your goal.

Reply:
@derrabus valid points, but IMO, it doesn't hurt, quite the contrary. It's not like I'm forcing a major upgrade.

Reply:
> I mean, composer will automatically pick 2.12 since it's the latest stable version.

Also, that's true if you run `composer update`, but not if you run `composer doctrine/doctrine-bundle`, and I know some people do that kind of thing.